beacon-chain/p2p/discovery.go

```diff
@@ -177,44 +182,27 @@ func (s *Service) createListener(
 	if err != nil {
 		return nil, errors.Wrap(err, "could not create local node")
 	}
-	if s.cfg.HostAddress != "" {
-		hostIP := net.ParseIP(s.cfg.HostAddress)
-		if hostIP.To4() == nil && hostIP.To16() == nil {
-			log.Errorf("Invalid host address given: %s", hostIP.String())
-		} else {
-			localNode.SetFallbackIP(hostIP)
-			localNode.SetStaticIP(hostIP)
-		}
-	}
-	if s.cfg.HostDNS != "" {
-		host := s.cfg.HostDNS
-		ips, err := net.LookupIP(host)
+
+	bootnodes := make([]*enode.Node, 0, len(s.cfg.Discv5BootStrapAddrs))
```

Comment:
Just a nit 
```suggestion
	bootNodes := make([]*enode.Node, 0, len(s.cfg.Discv5BootStrapAddrs))
```
But if you prefer it all lowercase then modify the one on line 188

Reply:
(I do not commit your change since there is some variables references which are not modified. I'll fix it.)